### DISCUSSION TITLE
what is the process for having social media links updated in the "You just edited OpenStreetMap!" sidebar ?

### DISCUSSION THREAD

**Discussion by julcnx** [openstreetmap/iD#8958] [Q&A] [answered]
After saving changes in ID, a left sidebar with links to the local community forum, and social media show up.

What is the process for having some of these links changed or added ?

![image](https://user-images.githubusercontent.com/2485196/152629963-2e5c6305-eeb4-49d5-8cf5-3840cd1b27f0.png)

**Comment by 1ec5** [comment, marked as answer]
These links are maintained by the [osm-community-index](https://github.com/osmlab/osm-community-index/) project. Issues and pull requests are welcome.
